semgrep_output_v1.atd

```diff
@@ -630,11 +634,13 @@ type skip_reason <ocaml attr="deriving show"> = [
   | Irrelevant_rule <json name="irrelevant_rule">
   | Too_many_matches <json name="too_many_matches">
   (* New in osemgrep *)
-  | Gitignore_patterns_match
+  | Gitignore_patterns_match (* TODO: use JSON lowercase for consistency *)
```

Comment:
It's a case of inconsistency. The problem with inconsistency is that people start asking questions, such as "why do some constructors start with a capital letter and not others?" (it could be while trying to debug something and inspecting the JSON data or when reading the ATD spec). Fewer questions saves time.